funnel/static/js/scripts.js

```diff
@@ -1,80 +1,299 @@
 
-function radioHighlight(radioName, highlightClass) {
-    var selector = "input[name='" + radioName + "']";
-    $(selector + ":checked").parent().addClass(highlightClass);
-    var handler = function() {
-        $(selector).parent().removeClass(highlightClass);
-        $(selector + ":checked").parent().addClass(highlightClass);
+window.Talkfunnel = {};
+
+if (Modernizr.localstorage) {
```

Comment:
Do we need this check? All the major browsers seem to [support](http://caniuse.com/#search=localstorage) localstorage.